jupyterlab_bigquery/jupyterlab_bigquery/tests/handlers_test.py

```diff
@@ -0,0 +1,13 @@
+import unittest
+from unittest.mock import Mock, MagicMock, patch
+
+from jupyterlab_bigquery import handlers
+
+
+class TestList(unittest.TestCase):
```

Comment:
Add a TODO to add tests here for these functions we get a chance:
  - list_projects, list_datasets, list_tables, list_models, get_dataset_details, get_table_details